homeassistant/components/energyzero/const.py

```diff
@@ -0,0 +1,20 @@
+"""Constants for the EnergyZero integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Final
+
+DOMAIN: Final = "energyzero"
+ATTRIBUTION: Final = "Data provided by EnergyZero"
```

Comment:
This is used only once, so, using a constant is not really needed in this case.

```suggestion
```

(I know many integrations still use this pattern, but that is more of a historical leftover)